.travis.yml

```diff
@@ -2,6 +2,8 @@ sudo: required
 dist: precise
 group: edge
 
+bundler_args: --retry 3
```

Comment:
This should not be here, we already discussed in another repo about this. Please refer to it to apply also in this repo the configuration we ended up to agree on.